### ISSUE TITLE
./install-all.sh find vdpm_install... No such file or directory

### ISSUE THREAD

**Issue by olokos** [vitasdk/vdpm#97]
I'm not sure if that's by design or if this actually is an issue?

`find: ‘/tmp/vdpm_install_libmad’: No such file or directory`
which is then repeated for literally every entry with `find`

```
find: ‘/tmp/vdpm_install_pib’: No such file or directory
Installing pib...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 13 59808   13  8271    0     0   8930      0  0:00:06 --:--:--  0:00:06  8930include/
include/pib.h
include/EGL/
include/KHR/
include/GLES2/
include/GLES2/gl2platform.h
include/GLES2/gl2.h
include/GLES2/gl2ext.h
include/KHR/khrplatform.h
include/EGL/eglplatform.h
include/EGL/egl.h
include/EGL/eglext.h
100 59808  100 59808    0     0  54514      0  0:00:01  0:00:01 --:--:--  296k
lib/
lib/libpib.a
success
```
With that said, it seems like the script worked for me, but each line that had `find` like in the example `No such file or directory`

Not sure if that's expected or an actual bug?

**Comment by nabetse00** [comment]
Yes it is expected since vdpm makes a tmp file when a package is downloaded to remember that it's been installed.

Since its a first install then then this file doesn't exist 
```console
find: ‘/tmp/vdpm_install_XXX’: No such file or directory
```
and vdpm installs it ...

This may cause some trouble if you mess with /usr/local/vitasdk manualy.

```console
ls  /tmp/vdpm_install_*
```
will list packages installed.

if you have some problems on update a possible solution is to change in ./vdpm script
```bash
CLEAN=0
```
to 
```bash
CLEAN=1
```
then
```
 ./vdpm <somepackage this doesnt mater>
```
It will erase all /tmp/vdpm_install_* files 

Go back to 
```bash
CLEAN=0
```
And test ...

**Comment by looni01** [comment]
I’m trying to install vdpm the first time and have the same problem with "/tmp/vdpm_install_jansson" . 
Sadly the recommended change in ./vdpm script did not fix it. :(

**Comment by isage** [comment]
This isn't bug or issue.

If you're having real problems with vdpm - that's not it.

**Comment by frangarcj** [comment]
Closing this — since the 2026.08 release, `vdpm` doesn't go through `install-all.sh` / `install-vitasdk.sh` anymore. That whole thing (the per-package tarballs, the `/tmp/vdpm_install_*` bookkeeping) is gone; installs now go through a pacman-based client with signed channel manifests and per-package hash verification, so this exact failure shouldn't be possible anymore.

If you're setting up VitaSDK today, `./bootstrap-vitasdk.sh` replaces everything this issue is about. If you hit something similar with the current installer, open a fresh issue with the actual output and I'll take a look.

Thanks for reporting it back when it happened.
